### DISCUSSION TITLE
DAI 2024-2025 - My picocli project - Aladin Iseni

### DISCUSSION THREAD

**Discussion by aladin-heig** [heig-vd-dai-course/heig-vd-dai-course#266] [Show and tell]
https://github.com/aladin-heig/hello-world

**Comment by ludelafo** [comment]
Everything looks good to me, thank you!

I've just noticed you put the `README.md` in the `src` directory. I think it you be better at the root level of the repository. Was it my mistake or did I miss something?

**Reply by aladin-heig** [reply]
Thank you. It was my mistake. I didn't know where to put it and in my mind the src directory was logic. From now I will put it in the root level.
